README.md

```diff
@@ -186,6 +187,31 @@ method takes one argument:
 
 2. `map` must return a value of the same Functor
 
+### Contravariant Functor
+
+1. `u.contramap(a => a)` is equivalent to `u` (identity)
+2. `u.contramap(x => f(g(x)))` is equivalent to `u.contramap(f).contramap(g)`
+(composition)
+
+#### `contramap` method
+
+```hs
+contramap :: Contravariant f => f a ~> (b -> a) -> f b
```

Comment:
If `Contravariant` is the name of the type class, should we remove "Functor" from the heading?